registry.go

```diff
@@ -16,57 +16,66 @@ package log
 
 import (
 	"sync"
+
+	"github.com/tetratelabs/telemetry"
 )
 
+// Scope is a named logger that can be configured independently
+type Scope interface {
```

Comment:
`Scope` confused me as I thought it was the typed `scope` level, not a Logger with a name.

Consider:

```suggestion
// ScopedLogger is a named logger that can be configured independently
type ScopedLogger interface {
```